faq.html

```diff
@@ -0,0 +1,155 @@
+                <h6>First you have to install them. When you do, you only need to run the emulator and it will display a <em>Game Select</em> Screen.</h6>
+            </div>
+            <h3>How to install games?</h3>
+            <div class="my-5">
+                <h6>
+                    Windows:<br />
+                    Right click on a <code>.vpk</code>, go to Properties and click "Change..." to associate <code>Vita3K.exe</code> with <code>.vpk</code> files. Double-clicking them should work after that. Just make sure to replace everything in the same path when updating Vita3K, or do the association process again for newer Vita3K builds.
+                    Alternatively, drag and drop games to <code>Vita3K.exe</code>.<br /><br />
+                    Windows/Linux/macOS:<br />
+                    Supply them as a command-line argument to the executable (the latter is necessary for Linux/Mac OS).
+                    Alternatively, install them manually by unzipping the <code>.vpk</code>  to Vita3K's home directory (see below).
+                    Note: If you dumped via MaiDump, the game will not be in a <code>.vpk</code>. Either zip it up (the contents, not the root folder) and (optionally) rename it to <code>.vpk</code>, or install it manually as described above.
+                </h6>
+            </div>
+            <h3>Where are games installed?</h3>
+            <div class="my-5">
+                <h6>
+                    They are installed in the Vita3K home directory, in <code><Vita3K_home_dir>/ux0/app/<Title_ID></code>.
+                    Defaults are:<br />
+                    Windows: <code>C:/Users/<username>/AppData/Roaming/Vita3K/Vita3k</code><br />
+                    Linux: <code>~/.local/share/Vita3K/Vita3K</code><br />
+                    You can change it by editing <code>config.yml</code>'s "<code>pref-path</code>" entry.
+                </h6>
+            </div>
+            <h3>Can I play multiplayer games online with real consoles or other Vita3K users?</h3>
+            <div class="my-5">
+                <h6>
+                    Playing commercial games online with real PlayStation Vita systems would require the user to connect to PlayStation Network which isn't very feasible due to obvious technical and legal limitations.
+                    That said, networking is partly implemented and you can play some <strong>homebrew</strong> multiplayer games like <a href="https://github.com/Rinnegatamante/vitaQuake">vitaQuake</a>just fine.
+                </h6>
+            </div>
+            <h3>How to use LLE (Low-Level Emulation) for OS modules?</h3>
+            <div class="my-5">
+                <h6>
+                    Go to <code>Help</code> -> <code>Welcome</code> -><code>Download Firmware</code>. This will decrypt the modules only.
+                    If you also need all firmware files, like fonts, go to <code>Help</code> -> <code>Welcome</code> -><code>Download Firmware Font Package</code>.
```

Comment:
It should probably be mentioned how users can turn the LLE modules on and off

Reply:
changed title to
Where to download firmwares